docs/content/providers/docker.md

```diff
@@ -145,6 +145,7 @@ You can specify which Docker API Endpoint to use with the directive [`endpoint`]
         - Accounting at container level, by exposing the socket on a another container than Traefik's.
           With Swarm mode, it allows scheduling of Traefik on worker nodes, with only the "socket exposer" container on the manager nodes.
         - Accounting at kernel level, by enforcing kernel calls with mechanisms like [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux), to only allows an identified set of actions for Traefik's process (or the "socket exposer" process).
+        - SSH public key authentication (SSH is supported with Docker > 18.09)
```

Comment:
Can we give an example of what this looks like?

is it `ssh://user@example.com:22`?

We provide an example for TCP, but we should for SSH too!

Reply:
Okay, will do, just a sec.

Reply:
@dtomcej  Where should that be demonstrated? also, you're right about URL, `ssh://u@h` is absolutely right.

Reply:
https://github.com/containous/traefik/blob/master/docs/content/providers/docker.md#endpoint

Reply:
@dtomcej as a comment?
```yaml tab="File (YAML)"
providers:
  docker:
    endpoint: "unix:///var/run/docker.sock"
    # endpoint: "ssh://user@example.com"
```

Reply:
Ahh, okay
![image](https://user-images.githubusercontent.com/19504461/70740364-0b155200-1d2a-11ea-9ec3-8e5e971729c1.png)

Reply:
@dtomcej Done!